lib/chef/knife/kvm_vm_create.rb

```diff
@@ -140,6 +144,15 @@ def read_and_validate_params
           exit 1
         end
 
+        if (config[:location].nil? &&
```

Comment:
can you use an XOR operator here? `config[:location].nil? ^ config[:iso_image].nil?`